src/test/bench/omap_create.cc

```diff
@@ -0,0 +1,214 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+#include <iostream>
+#include <vector>
+#include <map>
+#include <ctime>
+#include <chrono>
+
+#include <cfloat>
+
+#include <getopt.h>
+
+#include "global/global_init.h"
+#include "common/ceph_argparse.h"
+#include "include/rados/librados.h"
```

Comment:
also, i think `librados.hpp` is self-contained. so no need to include the C API.